File: src/mobbnb.js
'use strict';

class Mobbnb{
  constructor(){
    this._email = "";
    this._password = "";
  }
  addUserEmail(email) {
    this._email = email;
  }
  getEmail(){
    return this._email
  }
  addUserPassword(password) {
    this._password = password;
  }
  
}

File: lib/mobbnb.rb
class Mobbnb
  attr_reader :email, :password

  def initialize(email:, password:)
    @email = email
    @password = password
  end

  def self.add_email(email)
    @email = email
  end

  def self.add_password(password)
    @password = password
  end

end
